integration-tests/testsetups/ocr.go

```diff
@@ -139,18 +144,31 @@ func (o *OCRSoakTest) Setup(t *testing.T, env *environment.Environment) {
 	require.NoError(t, err, "Error waiting for OCR contracts to be deployed")
 	for _, ocrInstance := range o.ocrInstances {
 		o.ocrInstanceMap[ocrInstance.Address()] = ocrInstance
-		o.TestReporter.ContractReports[ocrInstance.Address()] = testreporters.NewOCRSoakTestReport(
-			ocrInstance.Address(),
-			o.Inputs.StartingAdapterValue,
-			o.Inputs.ExpectedRoundTime,
-		)
 	}
 	l.Info().Msg("OCR Soak Test Setup Complete")
 }
 
 // Run starts the OCR soak test
 func (o *OCRSoakTest) Run(t *testing.T) {
 	l := utils.GetTestLogger(t)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	latestBlockNum, err := o.chainClient.LatestBlockNumber(ctx)
+	cancel()
```

Comment:
For whatever reason this is breaking my brain right now. Just to verify if the ctx timeout occurs before you get the block number then you will get an error returned correct?

Reply:
If not you might need to make sure you have a valid block number after the fact as well as checking the error.

Reply:
> if the ctx timeout occurs before you get the block number then you will get an error

Correct.